src/pages/LogDrop/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { useCallback, useRef, useState } from "react";
+import styled from "@emotion/styled";
+import Button from "@leafygreen-ui/button";
+import { palette } from "@leafygreen-ui/palette";
+import { Option, Select } from "@leafygreen-ui/select";
+import { Body } from "@leafygreen-ui/typography";
+import { useDropzone } from "react-dropzone";
+import Icon from "components/Icon";
+import { LogTypes } from "constants/enums";
+import { size } from "constants/tokens";
+import { useLogContext } from "context/LogContext";
+
+const { red } = palette;
+interface LogDropProps {
+  onChangeLogType: (logType: LogTypes) => void;
+}
+
+const LogDrop: React.FC<LogDropProps> = ({ onChangeLogType }) => {
+  const { ingestLines } = useLogContext();
+  const [hasDroppedLog, setHasDroppedLog] = useState(false);
+  const [logType, setLogType] = useState(LogTypes.RESMOKE_LOGS);
```

Comment:
Since we're not doing any type of file detection, it feels more intuitive to me to default to no selection here and require users to choose the type of log they've uploaded.